foundations/foundation-ssl/src/test/java/io/servicecomb/foundation/ssl/SSLOptionTest.java

```diff
@@ -230,6 +230,46 @@ public void testSSLOptionYamlOption2() throws Exception {
     System.clearProperty("ssl.protocols");
   }
 
+  @Test
+  public void testSSLOptionYamlOptionWithProperyFalse() throws Exception {
+    System.setProperty("ssl.authPeer", "false");
+    DynamicConfiguration configFromYamlFile =
+        new DynamicConfiguration(yamlConfigSource(), new NeverStartPollingScheduler());
+    // configuration from system properties
+    ConcurrentMapConfiguration configFromSystemProperties =
+        new ConcurrentMapConfiguration(new SystemConfiguration());
+    ConcurrentCompositeConfiguration finalConfig = new ConcurrentCompositeConfiguration();
+    finalConfig.addConfiguration(configFromSystemProperties, "systemEnvConfig");
+    finalConfig.addConfiguration(configFromYamlFile, "configFromYamlFile");
+
+    SSLOption option = SSLOption.buildFromYaml("server", finalConfig);
+
+    boolean authPeer = option.isAuthPeer();
+    option.setAuthPeer(authPeer);
+    Assert.assertEquals(false, authPeer);
+    System.getProperties().remove("ssl.authPeer");
+  }
+
+  @Test
+  public void testSSLOptionYamlOptionWithProperyTrue() throws Exception {
+    System.setProperty("ssl.authPeer", "true");
+    DynamicConfiguration configFromYamlFile =
```

Comment:
it seems that finalConfig equals io.servicecomb.config.ConfigUtil.createLocalConfig() result?